tests/test_execution.py

```diff
@@ -1472,3 +1472,27 @@ def test_loop_else(harness: _TestHarness) -> None:
 
 def test_regression_194(harness: _TestHarness) -> None:
     harness.deploy(TEST_CASES_DIR / "regression_tests" / "issue_194.py")
+
+
+def test_inlining(harness: _TestHarness) -> None:
+    def test() -> None:
```

Comment:
This looks like a duplication of the test contract? You could deploy from the compilation result there, like in the test above. This won't actually test the method though - for that you'd probably want to add a test in `test_arc32.py` (which on reflection just now should probably be `test_excution_arc4.py` or similar, oh well).

Reply:
The extra test has been removed.